package.json

```diff
@@ -265,7 +265,7 @@
     "@fortawesome/fontawesome-free": "^5.12.1",
     "@hot-loader/react-dom": "^16.12.0",
     "amazon-chime-sdk-js": "^1.16.0",
-    "aws-amplify": "^2.2.6",
+    "aws-amplify": "^2.2.1",
```

Comment:
Downgraded? We wanted to bump it up right? Also can you once do `npm install` to update the `package-lock.json` as well.